code/controllers/subsystem/air.dm

```diff
@@ -84,6 +84,18 @@ SUBSYSTEM_DEF(air)
 /datum/controller/subsystem/air/fire(resumed = 0)
 	var/timer = TICK_USAGE_REAL
 
+	var/TiDi = round((Master.tickdrift/(world.time/world.tick_lag))*100,0.1)
+	switch(TiDi)
+		if(0 to 15)
+			wait = 5 //This isn't an initial() call because that could get nasty with how much this thing runs.
```

Comment:
Might be worth making this a define. Could even have the other wait times as multiplications of the define if you're a gamer 😎